src/main/scala/org/broadinstitute/dsde/workbench/avram/UserInfo.scala

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.dsde.workbench.avram
+
+import org.broadinstitute.dsde.workbench.avram.dataaccess.SamUserInfoResponse
+
+case class UserInfo(subjectId: String, email: String, enabled: Boolean, token: String)
+
+object UserInfo {
+  def apply(samUserInfo: SamUserInfoResponse, token: String): UserInfo =
+    UserInfo(
+      samUserInfo.userSubjectId,
+      samUserInfo.userEmail,
+      samUserInfo.enabled,
+      token)
+}
```

Comment:
Should this be in AvramModel.scala?

Reply:
It could, though I think of a system's model as the domain it controls... collections and entities in this case. `UserInfo` is just a convenience object to bundle the info from Sam with the token (for later queries to Sam). That said, I didn't mean to leave it in `org.broadinstitute.dsde.workbench.avram` but I also wasn't sure about putting it in `util` or making a new package. I'm open to suggestions.